src/components/recordedit/recordedit.tsx

```diff
@@ -297,7 +298,19 @@ const RecordeditInner = ({
     }
   }, [formProviderInitialized]);
 
+  // This useEffect triggers when addFormsEffect is set to true when "clone" is clicked
```

Comment:
Thank you for explaining it. I was confused why this is needed. Is this something worth documenting in our dev guides? As a bullet in React/TypeScript section where we're talking about hooks.

Also for readability I think it's better if we use jsdoc style of comments for these lengthy multi-line comments.
```
/**
 *
 */
```